tiny_cnn/io/tensorflow/proto_parser.h

```diff
@@ -0,0 +1,256 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <map>
+#include "tensorflow/core/framework/graph.pb.h"
+using namespace std;
+using namespace tensorflow;
+
+template<class in_value>
+void all2str(string & result, const in_value& t)
+{
+    ostringstream oss;
+    oss<<t;
+    result=oss.str();
+}
+
+static float char_to_float(const char* b) {
```

Comment:
I use this ad reference https://isocpp.org/wiki/faq/inline-functions#inline-and-perf